lib/core/app/controllers/cloud_event_subjects_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CloudEventSubjectsController < ApplicationController
+  skip_before_action :authenticate_it!, only: [:index]
+  skip_after_action :set_headers!, only: [:index]
+
+  def index
+    render json: json_resources(resource_class: CloudEventSubjectResource, records: resources)
+  end
+
+  private
+
+  def resources
+    path = "#{Settings.event_logging.config.schemas_path}/#{Settings.service.name}"
+    Dir.foreach(path).map do |file_name|
+      next unless file_name.end_with?('.avsc')
```

Comment:
use `Dir["#{path}/*.avsc"].each do |file_name|` and remove the `next` guard and the `compact`